docs/changes.rst

```diff
@@ -232,7 +232,7 @@ Version 1.0.10rc1 (2017-01-29)
 Bug fixes:
 
 - borg serve: fix transmission data loss of pipe writes, #1268
-  This affects only the cygwin platform (not Linux, *BSD, OS X).
+  This affects only the cygwin platform (not Linux, BSD, OS X).
```

Comment:
Meh, I just cherry pick these.

At the moment merging maintenance branches is just-so still viable. I think in the future we'll need to go for cherry picking stuff always, the past few merges were always getting messier and messier...